recipes/antismash/meta.yaml

```diff
@@ -37,31 +33,31 @@ requirements:
     - glimmer ==3.02
     - glimmerhmm ==3.0.4
     - helperlibs
-    - hmmer
+    - hmmer ==3.1b2
     - hmmer2
     - openjdk
     - libxml2 ==2.9.4
     - mafft
     - matplotlib
-    - meme ==4.11.2
+    - meme ==4.12.0
```

Comment:
I'd have to check `CASSIS` still works with that version of meme. I seem to recall that they changed the output file format from 4.11.3 to 4.11.4, and that broke some things. I don't remember if we got around to fixing it in a version-agnostic manner. I'll get back to you on that.
Is there a particular reason you're not leaving it pinned to 4.11.2, which definitely works?